src/platform/nodejs/lib/util/http.ts

```diff
@@ -219,15 +226,32 @@ const Http: typeof IHttp = class {
   }
 
   checkConnectivity = (callback: (errorInfo: ErrorInfo | null, connected?: boolean) => void): void => {
-    // TODO: fix this
+    if (this.options.disableConnectivityCheck) {
+      callback(null, true);
+      return;
+    }
+    const internetUpUrl = this.options.internetAvailabilityUrl || Defaults.internetUpUrl;
```

Comment:
Should we be consistent and call the default value `internetAvailabilityUrl` too?